insights/parsers/x86_debug_enabled.py

```diff
@@ -0,0 +1,33 @@
+"""
+Parsers for file ``/sys/kernel/debug/x86/*_enabled`` outputs
+============================================================
+
+This module provides the following parsers:
+
+X86Enabled - file ``/sys/kernel/debug/x86/*_enabled``
+------------------------------------------------------------
+"""
+
+from insights import Parser
+from insights.parsers import SkipException
+
+
+class X86DebugEnabled(Parser):
+    """
+    Class for parsing file ``/sys/kernel/debug/x86/*_enabled``
+
+    Attributes:
+        value (int): the result parsed of '/sys/kernel/debug/x86/*_enabled'
+
+    Raises:
+        SkipException: When input content is empty
+    """
+    def parse_content(self, content):
+        EMPTY = "Input content is empty"
+        if not content:
+            raise SkipException(EMPTY)
+
+        if str(content[0]).isdigit():
+            self.value = int(content[0])
+        else:
+            self.value = content[0]
```

Comment:
You have clarified that the `self.value` is an `int`, but here it might be a string.

Better raise a ParseException here.